tests/framing-helper.c

```diff
@@ -33,8 +37,10 @@ static void remove_framing()
 {
     static const size_t buffer_size = 4096;
     char *buffer = malloc(buffer_size);
-    if (!buffer)
-        err(EXIT_FAILURE, "malloc");
+    if (!buffer) {
+        fprintf(stderr, "malloc");
```

Comment:
The `err` call would print out the failure reason and the `fprintf` one doesn't. I can't imagine what we'd learn from the failure code, but it would be nice if the error message were `"malloc failed"` instead of just `"malloc"`.